tools/c7n_azure/c7n_azure/resources/sqldatabase.py

```diff
@@ -30,3 +39,176 @@ class resource_type(ArmResourceManager.resource_type):
             manager_name='sqlserver',
             annotate_parent=True
         )
+
+
+class BackupRetentionPolicyFilter(Filter):
+
+    # TODO: consolidate with MetricFilter.op?
```

Comment:
Core C7N has the full ops array as used in value filter, but in this case you don't want all the string/array support.  If we have multiple use cases of this list then yeah I think consolidate it into our filters.py or something.  `scalar_ops` or `number_ops` or whatever.

Reply:
moved this to `scalar_ops` in filters.py